std/mem.zig

```diff
@@ -8,6 +8,11 @@ const meta = std.meta;
 const trait = meta.trait;
 const testing = std.testing;
 
+pub const page_size = switch (builtin.arch) {
+    .wasm32, .wasm64 => 64 * 1024,
+    else => 4 * 1024,
```

Comment:
There is no longer just one page size; huge pages are getting used more and more.

Reply:
that's going to have to be a separate and even-more-breaking change if page size stops being comptime known. out of scope for this PR

Reply:
Created https://github.com/ziglang/zig/issues/2564 to track.